source/js/Module/Event/Components/EventItem.js

```diff
@@ -1,29 +1,74 @@
 import PropTypes from 'prop-types';
+var today = new Date();
 
-const EventItem = ({ event, gridColumn, displayFields }) => (
-    <a className="c-card c-card--action" href={event.permalink} style={{textAlign: 'center', height: '100%'}}>
-      {displayFields.includes('image') && event.image_url && (
-        <div className="c-card__image c-card__image--secondary">
-          <div className="c-card__image-background" style={{backgroundImage: `url('${event.image_url}')`}}></div>
-        </div>
-      )}
-
-      <div className="c-card__body">
-        {event.post_title && <h3 className="c-card__title">{event.post_title}</h3>}
+function getDateBadge(event, type) {
+  let startDate = parseInt(event.start_date.substr(0, 4) + event.start_date.substr(5, 2) - 1 + event.start_date.substr(8, 2));
```

Comment:
Using let but we are not overriding or changing it's value (https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/) change all let to const